docs/how-to/websites-on-ipfs/multipage-website.md

```diff
@@ -5,3 +5,145 @@ title: Multipage website
 # Multipage website
 
 In this guide you will learn how to deal with hosting a website with multiple pages and external assets on IPFS. We recommend that you follow through the [single page website guide](single-page-website) first.
+
+There are a few ways of creating multiple pages.
+
+- Individually upload files and link through CID.
+- Upload a directory with relative paths.
+
+## Indivudally upload files
+
+1. Create a file called `index.html` and paste the following code. We'll continue using the code from the [single page website](how-to/websites-on-ipfs/single-page-website/#add-your-site).
+
+2. Open IPFS desktop and upload `index.html`
+
+![](./images/multipage-website/upload-file.gif)
+
+3. Copy the link. We'll use this to link for the `about` page.
+
+4. Create a file called `about.html` and paste in the following code.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>About</title>
+    <meta
+      name="description"
+      content="Get a random fact about a planet in our solar system."
+    />
+    <meta name="author" content="The IPFS Docs team." />
+    <style>
+      body {
+        margin: 15px auto;
+        max-width: 650px;
+        line-height: 1.2;
+        font-family: sans-serif;
+        font-size: 2em;
+        color: #fff;
+        background: #444;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>About</h1>
+    <!-- Use the link from step 3 -->
+    <a href="https://ipfs.io/ipfs/$SITE_CID"> Link to Random Planet Facts </a>
+  </body>
+</html>
+```
+
+Notice how the `href` is an absolute path.
+
+5. Open IPFS desktop and upload `about.html`
+
+![](./images/multipage-website/upload-about.gif)
+
+6. Once uploaded, verify that the link works.
+
+![](./images/multipage-website/link-file.gif)
```

Comment:
Try to describe what's happening in each gif, if possible. I've done the previous two as an example.

Reply:
will do